grudge/symbolic/operators.py

```diff
@@ -103,8 +104,11 @@ class Operator(pymbolic.primitives.Expression):
 
     def __init__(self, dd_in, dd_out):
         import grudge.dof_desc as dof_desc
-        self.dd_in = dof_desc.as_dofdesc(dd_in)
-        self.dd_out = dof_desc.as_dofdesc(dd_out)
+        dd_in = dof_desc.as_dofdesc(dd_in)
+        dd_out = dof_desc.as_dofdesc(dd_out)
+
+        self.dd_in = dd_in
+        self.dd_out = dd_out
```

Comment:
```suggestion
        self.dd_in = dof_desc.as_dofdesc(dd_in)
        self.dd_out = dof_desc.as_dofdesc(dd_out)
```
? Unless I misunderstand why they were separated.

Reply:
Whoops! This was leftover from the previous `QTAG_NONE` hunting spree. I'll fix this!

Reply:
Reverted this (and similar no-op changes) in: https://github.com/inducer/grudge/pull/83/commits/360ed3a7853c5cf2ab24bce93d40d785f7933be6